docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
+3. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the assets:
 
-Open the `__init__.py` file, located in `/tutorial_template/tutorial_dbt_dagster`, and add the following code:
+   <Image
+   alt="Asset graph in Dagster's UI, containing dbt models loaded as Dagster assets"
+   src="/images/integrations/dbt/using-dbt-with-dagster/load-dbt-models-dbt-asset-graph.png"
+   width={1090}
+   height={894}
+   />
 
-```python
-import os
 
-from dagster_dbt import DbtCli
-from tutorial_dbt_dagster import assets
-from tutorial_dbt_dagster.assets import DBT_PROJECT_PATH
+---
 
-from dagster import Definitions, load_assets_from_modules
+## Step 3: Build your dbt project, aka materialize your assets
 
-resources = {"dbt": DbtCli(project_dir=DBT_PROJECT_PATH)}
+You can do more than view your dbt models in Dagster – you can also run them. In Dagster, running a dbt model corresponds to _materializing_ an asset. Materializing an asset means running some computation to update its contents in persistent storage. In this tutorial, that persistent storage is our local DuckDB database.
 
-defs = Definitions(assets=load_assets_from_modules([assets]), resources=resources)
-```
+To build your dbt project, i.e. materialize your assets, click the **Materialize all** button near the top right corner of the page. This which will launch a run to materialize the assets. When finished, the **Materialized** and **Latest Run** attributes in the asset will be populated:
 
-Let's take a look at what's happening here:
+<Image
+alt="Asset graph in Dagster's UI, showing materialized assets"
+src="/images/integrations/dbt/using-dbt-with-dagster/upstream-assets-asset-graph-materialized.png"
+width={1091}
+height={896}
+/>
 
-- In the `resources` key, we've provided configuration info for the <PyObject module="dagster_dbt" object="DbtCli" /> resource.
-- Added all assets in the assets module and the resources mapped to the `resources` key to the <PyObject object="Definitions" /> object. This supplies the resource we created to our assets.
-- Using <PyObject object="load_assets_from_modules" />, we've added all assets in the `assets` module as definitions. This approach allows any new assets we created to be automatically included in the code location instead of needing to manually add them one by one.
+After the run completes, you can:
+
+- Click the **asset** to open a sidebar containing info about the asset, including its last materialization stats and a link to view the [**Asset details** page](/concepts/dagit/dagit#asset-details)
+- Click the ID of the **Latest Run** - in the above image, that's `651489a2` - in an asset to view the [**Run details**](/concepts/dagit/dagit#run-details) page. This page contains detailed info about the run, including timing information, errors, and logs.
 
 ---
 
-## Step 3: View the assets in the Dagster UI
+## Step 4: Review the Python code in your Dagster project
```

Comment:
Is this the best name for this section? Sure we're reviewing code here, but what's the thing we want to achieve by the end of this part? I think going with something along those lines could be an improvement

Reply:
Do you think "Understand" would be better than "Review"?